src/constants/entities.asm

```diff
@@ -35,6 +35,19 @@ ENTITY_OPT1_B_IS_MINI_BOSS           equ 2
 ENTITY_OPT1_B_EXCLUDED_FROM_KILL_ALL equ 1
 ENTITY_OPT1_B_MOVE_PIT_WATER         equ 0
 
+; Values for wEntitiesPhysicsFlagsTable
+ENTITY_PHYSICS_HARMLESS             equ $80 ; The entity does not damage Link
+ENTITY_PHYSICS_PROJECTILE_NOCLIP    equ $40 ; Projectiles pass through the entity
+ENTITY_PHYSICS_PICKUP               equ $20 ; The entity can be grabbed with the Power Bracelet
```

Comment:
Pickup to me means more like a rupee, or an item. GRABBABLE might be a better name?

Reply:
Or maybe LIFTABLE?

Reply:
Yep, agreed!

Reply:
@daid @kemenaran I think "GRABBABLE" is better, because that seems to actually be what it is! I don't think this flag necessarily means that Link can lift it, from my limited testing. (I tried setting the flag on a few entities while playing, and were able to grab them but not lift them.)

Reply:
... But when I look at what entities have this flag set in `physics_flags.asm` I'm not so sure anymore. It's mostly things I wouldn't expect to be able to grab, including pickups such as rupees.

Reply:
If you remove the flags, can you still pickup rupees? It might be more something along the lines of "collision with link handled by entity code"?

Reply:
Should we just add a comment stating that "grabbable" might be a better name, merge this, and investigate further later?

Reply:
@kemenaran Maybe... I notice that my PRs tend to stall on naming issues 😅 There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.

I would love to test this more, but perhaps it should just be merged.

Reply:
Well, I feel that even with imperfections, this PR is an improvement. Plus having PRs stalled is never fun for the author – especially on hobby projects. 

So I’d say “let’s add a comment and merge it”. Do you have some bandwidth to add the comment, or do you want me to do it?

Reply:
Yep, I'll do it today.